prime-dotnet-webapi/prime-dotnet-webapi.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Is this file necessary?

Reply:
Not necessary, but VS code keep adding it back for some reason. I just let it through since it won't hurt anything, then I don't need to remove it every build.